assets/js/githubrepoRequstForm.js

```diff
@@ -1,8 +1,19 @@
 jQuery(function () {
+  $('#submitdata').on('click', async e => {
+    e.preventDefault(e)
+    console.log(`submit clicked`)
```

Comment:
So no need for any of this. The request is not going to come from here. The `<form>` tag has a `method` of `POST` and an `action` which is the URL/route you want to post to.

Reply:
file removed